bin/brew

```diff
@@ -103,19 +111,20 @@ then
              http_proxy https_proxy ftp_proxy no_proxy all_proxy HTTPS_PROXY FTP_PROXY ALL_PROXY \
              GITHUB_ACTIONS GITHUB_WORKSPACE GITHUB_ACTIONS_HOMEBREW_SELF_HOSTED \
              GITHUB_REPOSITORY GITHUB_RUN_ID GITHUB_SHA GITHUB_HEAD_REF GITHUB_BASE_REF GITHUB_REF \
-             "${!HOMEBREW_@}"
+             HOMEBREW_BREW_FILE HOMEBREW_EDITOR HOMEBREW_GITHUB_API_TOKEN HOMEBREW_LIBRARY HOMEBREW_PATH \
+             HOMEBREW_PREFIX HOMEBREW_REPOSITORY
```

Comment:
This is going to break a lot - like anything in `env_config.rb`.

Reply:
Yup. I'd just add a shellcheck ignore for this with a comment.